tests/testhelpers/generators.go

```diff
@@ -17,13 +17,15 @@ func NewRand() *rand.Rand {
 // Test Helpers that are useful for Generating random Scoot Api Structs
 // To help with testing the Scoot API and Scheduler
 
-// Generates a scoot.JobDefinition
-func GenJobDefinition(rng *rand.Rand) *scoot.JobDefinition {
-
+// Generates a scoot.JobDefinition with numTasks tasks (or random if numTasks == -1)
```

Comment:
I don't want to have a whole bunch of tests all throughout the code base that we have to try to keep synced. It seems like the server should implement that (because it's where we want backoff to be done)